docs/user_guide/HOWTO/Redundant_LDAP.rst

```diff
@@ -72,41 +75,40 @@ Configure the following settings in ``simp_def.yaml`` in the hiera directory:
   sssd::domains:
     - LDAP
 
-
-Add the `simp::ldap_server` class into the yaml file for the ldap server in
-Hiera (`hieradata/hosts/ldap_server1.your.domain.yaml`):
+Add the ``simp::server::ldap`` class into the yaml file for the LDAP server in
+Hiera, for example: ``hieradata/hosts/ldap_server1.your.domain.yaml``:
 
 .. code-block:: yaml
 
   classes :
     - 'simp::server::ldap'
 
-Leave any other classes that are there if they are needed.  Run the puppet
-agent on the ldap server until it runs cleanly. Run the agent on the puppet
-server.  Once all the other clients update against the Puppet server, they will
-be able to authenticate against the LDAP server.  Adding users and groups is
+Leave any other classes that are there if they are needed. Run the Puppet
+agent on the LDAP server until it runs cleanly. Run the agent on the Puppet
+server. Once all the other clients update against the Puppet server, they will
+be able to authenticate against the LDAP server. Adding users and groups is
 described in the :ref:`User_Management`.
 
 .. NOTE::
 
  Information on how the create salted ({SSHA}) passwords can be found at the
  `OpenLDAP site <http://www.openldap.org/faq/data/cache/347.html>`__.
 
-
-Set up the Redundant(Slave) Servers
------------------------------------
+Set up the Redundant (Slave) Servers
+------------------------------------
 
 Default Settings
 ~~~~~~~~~~~~~~~~
 
-Once the master is ready, LDAP slave nodes can be configured to replicate data
-from the master. These servers are read-only, and modifications cannot be made
-to LDAP entries while the master is down.
+Once the LDAP master is ready, LDAP slave nodes can be configured to replicate
+data from the master. These servers are read-only, and modifications cannot be
+made to LDAP entries while the master is down.
 
-Slave nodes can be configured via hiera by using `simp_options::simp::ldap_server::is_slave`,
-setting the replication id, and adding the `simp_options::simp::ldap_server` class.  This
-will set up your redundant server using the defaults. To do these three things,
-add the following lines to the
+Slave nodes can be configured via Hiera by setting
+``simp_options::simp::ldap_server::is_slave`` to ``true``, setting the
```

Comment:
simp_options::simp::ldap_server::is_slave should be simp::server::ldap::is_slave